.github/workflows/deploy.yml

```
@@ -46,20 +46,82 @@ jobs:
+        python -m pip install 'dvc[gcs]'
+        echo "Pulling puzzle data from DVC..."
+        cd scripts/datascience/themes_quality/wiki_puzzle_gemini_pipeline
+        dvc pull data/outputs/gemini-puzzles_firebase.json data/outputs/gemini-puzzles.json
+        echo "DVC puzzle data pulled successfully"
+        echo "Available puzzle files:"
+        ls -la data/outputs/
+
+    - name: Run Database Migration - Upload Gemini Puzzles
+      run: |
+        echo "Discovering puzzle files..."
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)